samples/msal-node-samples/client-credentials-with-cert-from-key-vault/test/client-credentials-with-cert-from-key-vault-aad.spec.ts

```diff
@@ -0,0 +1,96 @@
+import {
+    RETRY_TIMES,
+    validateCacheLocation,
+    NodeCacheTestUtils,
+} from "e2e-test-utils";
+import { ConfidentialClientApplication } from "@azure/msal-node";
+import config from "../config/AAD.json";
+import { getKeyVaultSecretClient } from "e2e-test-utils/src/KeyVaultUtils";
+import { getCertificateInfo } from "e2e-test-utils/src/CertificateUtils";
+import { LAB_CERT_NAME, LAB_KEY_VAULT_URL } from "e2e-test-utils/src/Constants";
+
+const TEST_CACHE_LOCATION = `${__dirname}/data/aad.cache.json`;
+
+const getClientCredentialsToken = require("../index");
+
+const cachePlugin = require("../../cachePlugin.js")(TEST_CACHE_LOCATION);
+
+const clientCredentialRequestScopes = ["https://graph.microsoft.com/.default"];
+
+describe("Client Credentials AAD Prod Tests", () => {
+    jest.retryTimes(RETRY_TIMES);
+    jest.setTimeout(90000);
+
+    let thumbprint: string;
+    let privateKey: string;
+    let x5c: string;
+    beforeAll(async () => {
+        await validateCacheLocation(TEST_CACHE_LOCATION);
+
+        const keyVaultSecretClient = await getKeyVaultSecretClient(
+            LAB_KEY_VAULT_URL
+        );
+        [thumbprint, privateKey, x5c] = await getCertificateInfo(
+            keyVaultSecretClient,
+            LAB_CERT_NAME
+        );
+
+        // TODO // get these from Lab Client, or am I allowed to save these as constants?
+        config.authOptions.clientId = "microsoft client id";
+        config.authOptions.authority = "microsoft authority";
```

Comment:
CC @hectormmg I was unable to get these values from the LabClient. It returns different values than the ones that I want.

Can I hardcode these as constants, or should I try to get from the LabClient?

I'll ping you regarding the values I'm talking about.